### PR TITLE
Improve style

### PR DESCRIPTION
**Pull request by mosabua**
## What are you changing in this pull request and why?

Change to active voice and present tense following the style guide.

## Checklist

- [x] Verified locally
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.

### CODE REVIEW

File: website/snippets/connect-starburst-trino/roles-starburst-enterprise.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [comment: For context, the section title used for this snippet is "Roles in Starburst Enterprise" ]: # 
 
-When connecting to a Starburst Enterprise cluster with built-in access controls enabled, you won't be able to provide the role as a suffix to the username, so the default role for the provided username will be used instead.
+When connecting to a Starburst Enterprise cluster with built-in access controls enabled, you are not able to provide the role as a suffix to the username, so the default role for the provided username is used instead.
```
Comment (nghi-ly) [inline website/snippets/connect-starburst-trino/roles-starburst-enterprise.md]:
```suggestion
If connecting to a Starburst Enterprise cluster with built-in access controls enabled, you can't add the role as a suffix to the username, so the default role for the provided username is used instead.
```

**Review by nghi-ly — COMMENTED** [review]
thanks for reviewing @mosabua ! left a small sugg

File: website/snippets/connect-starburst-trino/roles-starburst-enterprise.md (hunk 1 above)
Comment (mosabua) [reply website/snippets/connect-starburst-trino/roles-starburst-enterprise.md]: Updated

**Comment by mosabua** [comment]
Updated and rebased

**Review by nghi-ly — APPROVED** [review]
🚢

**Pull request merged**